lua/lazy/manage/task/git.lua

```diff
@@ -42,7 +42,12 @@ M.log = {
         error("no target commit found")
       end
       assert(target.commit, self.plugin.name .. " " .. target.branch)
-      if not Git.eq(info, target) then
+      if Git.eq(info, target) and Config.options.checker.check_pinned then
+        local last_commit = Git.get_commit(self.plugin.dir, target.branch, true)
+        if not Git.eq(info, { commit = last_commit }) then
+          self.plugin._.outdated = true
+        end
+      else
```

Comment:
I think you got your `if/then` wrong again here :)

When we are at the target, and `check_pinne=false`, it will now always show there's updates.

Move the `check_pinned` check inside the `if Git.eq(info, target) then` to make it work

Reply:
Wow this is embarrassing 😅 I swear I do know the basics of boolean logic.

Reply:
Too funny 😄